### PR TITLE
TASK-35540: Add style of external invitations list

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less [hunk 1]
```diff
@@ -55,8 +55,66 @@
 }
 
 .spaceInvitationDrawer {
-  .identitySuggester {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less:58]: Why identitySuggester is removed ?

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less [hunk 2]
```diff
@@ -55,8 +55,66 @@
...
+    background-color: @errorColorLight!important;
+    border: 2px solid @errorColor!important;
+    position: absolute;
+    z-index: 1;
+    width: 100%;
+    opacity: 0.9;
+    border-radius: 6px;
+    font-size: 0.875rem!important;
+  }
+  .successfulInvitation {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less:114]:
```suggestion
  .successfullInvitation {
```

File: platform-ui-skin/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 3]
```diff
@@ -456,6 +456,14 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <css-priority>1</css-priority>
   </portlet-skin>
 
+  <portlet-skin>
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/WEB-INF/gatein-resources.xml:459]: Is this forgotten ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less (hunk 1 above)
Comment (AzmiTouil) [reply platform-ui-skin/src/main/webapp/skin/less/social/portlets/SpaceMembers/Style.less:58]: useless code!

File: platform-ui-skin/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 3 above)
Comment (AzmiTouil) [reply platform-ui-skin/src/main/webapp/WEB-INF/gatein-resources.xml:459]: Yes !

**Review by azayati — DISMISSED** [review]
LGTM

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**